src/aics-image-viewer/components/App/index.tsx

```diff
@@ -340,42 +310,20 @@ export default class App extends React.Component<AppProps, AppState> {
     return makeChannelIndexGrouping(channels, viewerChannelSettings);
   }
 
-  stopPollingForImage(): void {
```

Comment:
for some background/context:  In a very early implementation, this viewer used to be able to request an image, and have that image returned by a service that could on-demand convert the file to a downsampled version. (The downsampled version would then be cached on the server after that first conversion) But that processing could take time and so the app would continuously poll in case of errors,crashes etc.